internal/api/v1/serviceclasses.go

```diff
@@ -0,0 +1,53 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/epinio/epinio/helpers/kubernetes"
+	"github.com/epinio/epinio/internal/cli/clients"
+	"github.com/epinio/epinio/internal/services"
+	"github.com/julienschmidt/httprouter"
+)
+
+type ServiceClassesController struct {
+}
+
+func (scc ServiceClassesController) Index(w http.ResponseWriter, r *http.Request) {
+	params := httprouter.ParamsFromContext(r.Context())
+	org := params.ByName("org")
+
+	cluster, err := kubernetes.GetCluster()
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	gitea, err := clients.GetGiteaClient()
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+
+	exists, err := gitea.OrgExists(org)
+	if handleError(w, err, http.StatusInternalServerError) {
+		return
+	}
+	if !exists {
+		http.Error(w, fmt.Sprintf("Organization '%s' does not exist", org),
+			http.StatusNotFound)
+		return
+	}
+	serviceClasses, err := services.ListClasses(cluster)
```

Comment:
Note here how listing the service classes does not make use of the org information at all.

Reply:
Done.